11-http/api/ctrls/users.go

```diff
@@ -50,34 +52,34 @@ func (uc usersCtrl) Retrieve(c *app.Context) {
 	if err != nil {
 		switch err {
 		case services.ErrInvalidID:
-			c.RespondBadRequest400(err)
+			c.RespondError(err.Error(), http.StatusBadRequest)
 			log.Println(c.SessionID, ": ctrls : Users : Retrieve : Completed : 400 :", err)
 
 		case services.ErrNotFound:
-			c.RespondNotFound404()
+			c.RespondError(err.Error(), http.StatusNotFound)
 			log.Println(c.SessionID, ": ctrls : Users : Retrieve : Completed : 404 : Not Found")
 
 		default:
-			c.RespondInternal500(err)
+			c.RespondError(err.Error(), http.StatusInternalServerError)
 			log.Println(c.SessionID, ": ctrls : Users : Retrieve : Completed : 500 :", err)
 		}
 
 		return
 	}
 
-	c.RespondSuccess200(&u)
```

Comment:
`u` is already a pointer to something. `&u` adds unnecessary indirection, and JSON marshalling doesn't even need a pointer in the first place.